From bb9d4d0c915854a2f39b3626d672b082208f8196 Mon Sep 17 00:00:00 2001
From: "Ding,XinX" <xinx.ding@intel.com>
Date: Wed, 21 Mar 2018 11:09:27 +0800
Subject: [PATCH 1139/2302] trusty: Update macro SMC_FC_GET_WALL_SIZE from 12
 to 20

Keep this macro synced with that of Trusty OS because we rebased
 trusty OS with Google's and this smc id was increased.

Change-Id: I22835c572a1998468fb4959f72856aebc51ac310
Signed-off-by: Ding,XinX <xinx.ding@intel.com>
---
 include/linux/trusty/smcall.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/trusty/smcall.h b/include/linux/trusty/smcall.h
index ee5dda2..3ab2f68 100644
--- a/include/linux/trusty/smcall.h
+++ b/include/linux/trusty/smcall.h
@@ -139,7 +139,7 @@
  * SMC_SC_DESTROY_WALL - notifies secure side that previously specifies SM Wall
  * object should be released usually as part of normal shutdown sequence.
  */
-#define SMC_FC_GET_WALL_SIZE	SMC_FASTCALL_NR(SMC_ENTITY_SECURE_MONITOR, 12)
+#define SMC_FC_GET_WALL_SIZE	SMC_FASTCALL_NR(SMC_ENTITY_SECURE_MONITOR, 20)
 #define SMC_SC_SETUP_WALL	SMC_STDCALL_NR(SMC_ENTITY_SECURE_MONITOR,  12)
 #define SMC_SC_DESTROY_WALL	SMC_STDCALL_NR(SMC_ENTITY_SECURE_MONITOR,  13)
 
-- 
2.7.4

